gradle/libs.versions.toml

```
@@ -81,10 +83,12 @@ detekt-formatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting",
 
 simple-java-mail = { module = "org.simplejavamail:simple-java-mail", version.ref = "simple-java-mail-version" }
 handlebars = { module = "com.github.jknack:handlebars", version.ref = "handlebars-version" }
+kotlinx-serialization-json = { module = "org.jetbrains.kotlinx:kotlinx-serialization-json", version.ref = "kotlinx-serialization-json-version" }
 
 [plugins]
 kotlin-jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin-version" }
 detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "detekt-version" }
 kotlinx-kover = { id = "org.jetbrains.kotlinx.kover", version.ref = "kotlinx-kover-version" }
 protobuf = { id = "com.google.protobuf", version.ref = "protobuf-version" }
 shadow-jar = { id = "com.gradleup.shadow", version.ref = "shadow-jar-version" }
+kotlinx-serialization = { id = "org.jetbrains.kotlin.plugin.serialization", version.ref = "kotlinx-serialization-version" }
```

```suggestion
kotlin-serialization = { id = "org.jetbrains.kotlin.plugin.serialization", version.ref = "kotlin-version" }
```

Updated with version and removed 'x'.